docs/admin-api-functions.md

```diff
@@ -46,7 +46,7 @@ pulsar-admin functions create \
     --inputs test-input-topic \
     --output persistent://public/default/test-output-topic \
     --classname org.apache.pulsar.functions.api.examples.ExclamationFunction \
-    --jar /examples/api-examples.jar
+    --jar $PWD/examples/api-examples.jar
```

Comment:
Is $PWD a safe assumption as to the proper location of the artifact, or should we use a more explicit nomenclature such as `path/to/examples/api-examples.jar` ?

Reply:
I think `$PWD` gives a good hint that it's an absolute path. At least I have found it useful with `pulsar-admin` when I have the files in the current directory and I want to reference the files with an absolute path. I guess there are also other ways.  Perhaps it could be explained to the user that `$PWD` is the current working directory. They will also find out if they try it or google it. :) 
In this particular case, the original `/examples/api-examples.jar` doesn't seem to make much sense. That's why I added the `$PWD` prefix.

Reply:
The `/path/to/file.jar` type of examples have the problem that for the user, it's really a lot of extra steps to find out the absolute path for the file arguments. That's why I prefer `$PWD/file.jar` since it's simple and if someone doesn't already know it, they have the chance to learn it. 

`$PWD` is for a unix shell, but the examples are already assuming a unix shell since the syntax is all unix like.